Add obj-mode recipe

abend opened the pull request:
Hi.  

Here is a recipe for my small obj-mode, which is for manually editing Wavefront 3D .obj text files.  

Let me know if any changes are required.

Thanks!

purcell commented:
Thanks for this!

The `obj` extension is used by a variety of systems, so `wavefront-obj-mode` would be a better name. Also, it's probably not advisable to `autoload` the `auto-mode-alist` entry for `.obj` for the same reason.

P.S. For [this line](https://github.com/abend/obj-mode/blob/master/obj-mode.el#L64) you can just use `ignore-errors`.

abend commented:
Hi.  Thanks for your help!

I've renamed the repo to wavefront-obj-mode.  Let me know if you'll need a new pull request for it.

I've also removed the sneaky `auto-mode-alist` modification and changed the `condition` usage to `ignore-errors` as you suggest.

Let me know if there's anything else I can do.

purcell commented:
Thanks. Added in d48e4fd. Sorry about the delay.
